src/stactools/package/commands.py

```diff
@@ -0,0 +1,68 @@
+import click
+import logging
+import json
+import os
+import requests
+
+logger = logging.getLogger(__name__)
+
+
+def read_metadata_json(source: str) -> dict:
+    """Collect a dict from a local or remote json source.
+
+    Enables the flexible use of json metadata files in CLI commands.
+
+    Args:
+        source (str): Path or url to a .json file
+
+    Returns:
+        dict: The contents of the json file converted to a dictionary
+    """
+    if os.path.isfile(source):
+        with open(source) as f:
+            data = json.load(f)
+    else:
+        metadata_response = requests.get(source)
+        data = metadata_response.json()
+
+    return data
+
+
+def create_stactoolspackage_command(cli):
+    """Creates the stactools-package command line utility."""
+    @cli.group(
+        "stactoolspackage",
+        short_help=("Commands for working with stactools-package"),
+    )
+    def stactoolspackage():
+        pass
+
+    @stactoolspackage.command(
+        "create-collection",
+        short_help="Creates a STAC collection",
+    )
+    @click.option(
+        "-d",
+        "--destination",
+        required=True,
+        help="The output directory for the STAC json",
+    )
+    def create_collection_command(destination: str):
+        """Creates a STAC Collection
+
+        Args:
+            destination (str): A placeholder to use click and demonstrate arguments
+        """
+        assert destination
+
+        raise NotImplementedError(
+            "The create-collection command has not been developed")
+
+    @stactoolspackage.command("create-item", short_help="Create a STAC item")
+    def create_item_command():
```

Comment:
```suggestion
    def create_item_command(source: str, destination: str) -> Item:
```

I know we don't enforce typing for all stactools stuff, but I think we should for the template to let people easily know what they're expected to do. Also, the "one file in, one file out" pattern is probably worth presenting for `create-item`.